packages/taquito-michel-codec/src/micheline-emitter.ts

```diff
@@ -12,7 +12,7 @@ export interface FormatOptions {
 }
 
 class Formatter {
-  constructor(private opt?: FormatOptions, private lev: number = 0) {}
```

Comment:
was this done by the linter? I think the previous one is already correct

Reply:
i think its my settings where the `{}` want a `{ }` same as any `{ key: 'string'}` instead of `{key: 'string'}`